solana/program_v2/packages/tests/src/gatekeeper-suite/set-gatekeeper-state.test.ts

```diff
@@ -94,8 +91,10 @@ describe('Gateway v2 Client', () => {
       gatekeeperAccount = await networkService.getGatekeeperAccount();
       const newState = gatekeeperAccount?.state;
 
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
       //@ts-ignore // Expects initial state of gatekeeper to be 0 (Active)
```

Comment:
This EnumMapper makes a whole lot of problems. I think there should be a better way. can we not use any `as T` in the mapper? or generics?